bundles/org.eclipse.ui.editors/src/org/eclipse/ui/texteditor/AbstractDecoratedTextEditor.java

```diff
@@ -330,6 +344,12 @@ public void gotoMarker(IMarker marker) {
 	 * @since 3.5
 	 */
 	private FocusedInformationPresenter fInformationPresenter;
+	/**
+	 * The handler for sticky scrolling
+	 * 
+	 * @since 4.3 //TODO ?
```

Comment:
So not I've found what I was looking for. This must be bundle major.minor version, which brings us to the next issue: this is new API and that means, bundle version must be updated from `3.17` to `3.18` and this is what should be added here instead of `4.3`.

Reply:
Done